pennylane/math/quantum.py

```diff
@@ -471,13 +472,20 @@ def vn_entropy(state, indices, base=None, check_state=False, c_dtype="complex128
 
     **Example**
 
+    You can get the entropy of a subsystem for any state vectors. You can find an example for the
+    maximally mixed state, where the entropy is maximal (default base for log is exponential).

```

Comment:
I think you mean maximally entangled state here (not maximally mixed).

Reply:
I think the whole system is maximally entangled, and the subsystem maximally mixed, right?